src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -558,9 +565,10 @@ Accept wildcard characters: False
 ```
 
 ### -VolumeType
-Specifies the type of virtual machine volumes to perform the encryption operation.
-Allowed values for virtual machines that run the Windows operating system are as follows: All, OS, and Data.
-The allowed values for Linux virtual machines are as follows: Data only.
+Specifies the type of virtual machine volumes on which to perform the encryption operation.
+Allowed values for Windows virtual machines: All, OS, and Data.
+The only allowed value for Linux virtual machines: Data.
```

Comment:
For VMSS, only Data is supported. 

For single Iaas VM's - Linux distributions that support OS disk encryption can have ALL (OS +Data), OS, or DATA 
Linux distributions that only support data disk encryption can only have VolumeType = DATA 
The list of volume types supported is currently maintained here:  https://docs.microsoft.com/en-us/azure/security/azure-security-disk-encryption-faq#bkmk_LinuxOSSupport